howso/engine/trainee.py

```diff
@@ -1101,7 +1101,7 @@ def predict(
         suppress_warning: Optional[bool] = False,
         use_case_weights: Optional[bool] = False,
         weight_feature: Optional[str] = None,
-    ) -> list:
+    ) -> DataFrame:
```

Comment:
is this correct though? It should only be a DataFrame when the client is the PandasClient I assume.

Reply:
Might not be worth worrying about too deeply at this point, but I think there are situations where we should expect a list of lists to come out of this method.

Reply:
Does this allow series responses? Also need to quote "DataFrame" here since it is imported under TYPE_CHECKING

Reply:
I'm going to remove this change as it was not part of the original PR.